tests/integration/run.bats

```diff
@@ -169,3 +169,20 @@ function docker_node_nabla_run() {
 	[[ "$output" == *"XXXXXXXXXX"* ]]
 	[ "$status" -eq 0 ]
 }
+
+@test "test memory runnc" {
+	local-test
+
+	# Check that 1024m is passed correct to runnc as 1024.
+	# Redirecting stderr to dev/null because there is a kernel warning
+	memory_check() {
+	  sudo docker run -d --rm --runtime=runnc -m 1024m nablact/nabla-node:test /hello/app.js 2>/dev/null
+	}
+
+	run memory_check
+	container_id=${output}
+	container_pid=$(docker inspect --format '{{.State.Pid}}' ${container_id})
+	run bash -c "sudo ps -e -o pid,command | grep ${container_pid}"
+	[[ "$output" == *"--mem=1024"* ]]
+	[ "$status" -eq 0 ]
```

Comment:
This check should be before using the `$output` so that it validates that `output` is indeed the expected STDOUT.

Reply:
I'm a little confused here. I have 2 different run statements to grab output. From my understanding with bats each one overwrites the global output. In this case they have no overlap as it's the container ID and the command line output.

Are you looking to have the status check moved before the output check? That's how the other tests are so I conformed to that.